packages/bpk-component-floating-notification/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bpk-component-floating-notification",
+  "version": "0.0.1",
+  "description": "Backpack floating-notification component.",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Skyscanner/backpack.git"
+  },
+  "author": "Backpack Design System <backpack@skyscanner.net>",
+  "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "bpk-component-button": "^6.2.5",
+    "bpk-component-text": "^7.0.4",
+    "bpk-mixins": "^31.1.1",
+    "bpk-react-utils": "^4.1.2",
+    "prop-types": "^15.5.8"
+  },
+  "peerDependencies": {
+    "react": "^16.3.0 || ^17.0.0"
```

Comment:
I think this will need to be `^16.8.0 || ^17.0.0` given we are using hooks in this component

Reply:
I'll get that updated